sample_data/zeiss_elta_r55/zeiss_elta_r55-REC_500.tops

```diff
@@ -1,129 +1,129 @@
-   0001 OR.COOR                                                                

```

Comment:
What's going on with these changes? Is this caused by newlines? I think it's better to leave these untouched.

Reply:
Nothing, I did not touch this file, I just move it.
I think it is the automatic change of git for line ending !

Reply:
I tried all my settings in Git for line endings and... those 2 files are always converted !

Reply:
Done !